Sources/PackageSigning/SignatureProvider.swift

```diff
@@ -113,6 +112,27 @@ public enum SignatureFormat: String {
             return CMSSignatureProvider(format: self)
         }
     }
+
+    public func privateKey(derRepresentation: Data) throws -> Certificate.PrivateKey {
+        switch self {
+        case .cms_1_0_0:
+            // TODO: don't need this check after https://github.com/apple/swift-package-manager/pull/6138
+            #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+            if #available(macOS 11.0, iOS 14.0, watchOS 7.0, tvOS 14.0, *) {
```

Comment:
@neonichu Do you know how far away we are from merging https://github.com/apple/swift-package-manager/pull/6138?